walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,126 @@
+                Event_Start.add(start);
+                Event_End.add(end);
+            }
+
+        if (event_num == 1) { 
+            Event only_event = events.iterator().next();
+            only_event_attendee = only_event.getAttendees().iterator().next();
+            String event_requester = request_attendees.iterator().next().toString();
+
+            if (event_requester != only_event_attendee) { // ignoresPeopleNotAttending
+                Collection<TimeRange> result = Arrays.asList(TimeRange.WHOLE_DAY);
+                return result;
+            }
+
+            else { // eventSplitsRestriction
+            	Collection<TimeRange> result = Arrays.asList(TimeRange.fromStartEnd(TimeRange.START_OF_DAY, Event_Start.get(0), false), 
+            	                                        TimeRange.fromStartEnd(Event_End.get(0), TimeRange.END_OF_DAY, true));
+                return result;
+            }
+        }
+
+        else if (event_num == 2) { 
+        	event_1 = Event_Times.iterator().next();
+            event_2 = Event_Times.iterator().next();
+            int free_time = Event_Start.get(1) - Event_End.get(0);
+
+        	if (Event_Start.get(0) == 480) { // everyAttendeeIsConsidered
+            	Collection<TimeRange> result = Arrays.asList(TimeRange.fromStartEnd(TimeRange.START_OF_DAY, Event_Start.get(0), false), 
+    	                                           TimeRange.fromStartEnd(Event_End.get(0), Event_Start.get(1), false),
+            	                                   TimeRange.fromStartEnd(Event_End.get(1), TimeRange.END_OF_DAY, true));
+            	return result;
+            }
+
+            else if (current_duration == 60) { // notEnoughRoom
+                Collection<TimeRange> expected = Arrays.asList();
+            }
+
+            else if (Event_End.get(0) == 510 && Event_Start.get(1) == 540) { // justEnoughRoom
+                Collection<TimeRange> result = Arrays.asList(TimeRange.fromStartDuration(Event_End.get(0), 30));
+                return result;
+            }
+
+            else if (Event_End.get(1) < Event_End.get(0)) { // nestedEvents
+                Collection<TimeRange> result = Arrays.asList(TimeRange.fromStartEnd(TimeRange.START_OF_DAY, Event_Start.get(0), false),
+            	                                    TimeRange.fromStartEnd(Event_End.get(0), TimeRange.END_OF_DAY, true));
+                return result;
+            }
+
+            else if (event_1.overlaps(event_2)) { // overlappingEvents
+             	Collection<TimeRange> result = Arrays.asList(TimeRange.fromStartEnd(TimeRange.START_OF_DAY, Event_Start.get(0), false),
+                	                                TimeRange.fromStartEnd(Event_End.get(1), TimeRange.END_OF_DAY, true));
+                return result;
+            } 
+        }
+    }
+    return Arrays.asList();
   }
+
+	public static int eventStart(TimeRange event_time) {
+        start = event_time.start();
+    	return start;
```

Comment:
The Java interpreter doesn't mind the whitespaces. However, please keep good format to ensure better readability.

We can do this:
public static int eventStart(TimeRange event_time) {
        start = event_time.start();
    	return start;
}

Or simply:

public static int eventStart(TimeRange event_time) {
        return event_time.start();
}